app/channels/game_channel.rb

```diff
@@ -0,0 +1,17 @@
+class GameChannel < ApplicationCable::Channel
+  def subscribed
+    return reject unless params[:game_id].present?
+
+    if game = Game.find_by(id: params[:game_id])
+      stream_for game
+      players = game.players.as_json(except: [:created_at, :updated_at])
+      GameChannel.broadcast_to(game, { player_list: players, game_started: game.started })
```

Comment:
Makes sense to format it before interacting in order to be reliable no matter the format that was being returned, very smart solution and thank you for explaining that in the PR description!